test/test.js

```diff
@@ -123,3 +123,69 @@ describe('dvb.find "Zellescher Weg"', function() {
         });
     });
 });
+
+describe('dvb.pins "5654791, 4620310, 5657216, 4623119, stop"', function() {
+    
```

Comment:
I'd love too, but as of now I couldn't figure out how to stub the newly introduced request-promise library. Ideas are very welcome!